front/app/components/InitiativeMap/components/InitiativeInformationOverlay.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { createPortal } from 'react-dom';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+import eventEmitter from 'utils/eventEmitter';
+import InitiativePreview from './InitiativePreview';
+import CloseIconButton from 'components/UI/CloseIconButton';
+import styled from 'styled-components';
+
+type Props = {
+  modalPortalElement: HTMLDivElement;
+};
+
+const StyledCloseButton = styled(CloseIconButton)`
+  width: 100%;
+  justify-content: flex-end;
+  padding-top: 8px;
+  padding-right: 8px;
+`;
+
+const InitiativeInformationOverlay = ({ modalPortalElement }: Props) => {
+  const [selectedInitiativeId, setSelectedInitiativeId] = useState<
```

Comment:
Maybe this state could also be in the URL search params

Reply:
I could do that - what do you think is best though? :thinking: I feel like I normally expect this information in state variables when looking at code.